CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 - PR #1713 Add documentation for Dask-XGBoost
 - PR #1666 CSV Reader: Improve performance for files with large number of columns
 - PR #1725 Enable the ability to use a single column groupby as its own index
+- PR #1767 Speed up unit tests
```

Comment:
Looks like this is a duplicate entry.  See further down the list.  Should we remove PR 1767 here?

Reply:
Looks like it was from the recent merge. Fixed, thanks!